tests/templates/test_state_preparations.py

```diff
@@ -392,6 +396,7 @@ def circuit(weights):
 
         assert np.allclose(qubit_device_3_wires.state, GHZ_state, atol=tol, rtol=0)
 
+    @pytest.mark.usefixtures("tape_mode")
     def test_even_superposition_generation(self, qubit_device_3_wires, tol):
         """Test that the template prepares a even superposition state."""
         even_superposition_state = np.ones(8)/math.sqrt(8)
```

Comment:
How come the global fixture is not used here?

Reply:
I noticed that most of the tests were low level and would be repeated unnecessarily. Worth still having it global just for maintenance?